src/Error/Error.js

```diff
@@ -3,7 +3,7 @@ import './Error.css';
 
 const Error = ({ error }) => {
   return(
-    <h1>{error}</h1>
+    <h1 className="error-message">{error}</h1>
```

Comment:
We tried adding some text here, but then our test broke. Maybe we can come back to this! :)